### PR TITLE
Basic tailwind integration

### PR DESCRIPTION
**Pull request by razzeee**
Adding taliwind and using it for all previous inline styles and some other fixes.

Should fix https://github.com/flathub/website/issues/300 and https://github.com/flathub/website/issues/310

Some things:
- Changes paddings and font sizes slightly
- Lists need to marked up (via a class) as lists, if we want bullets. (https://tailwindcss.com/docs/preflight) Not really a problem, but might be something we need to fix for release notes.
- ~~This doesn't touch colors for now~~
- This also slightly changes how the spinner works, hopefully for the better (I still think it's to much choice)

### CODE REVIEW

**Review by kymckay — APPROVED** [review]
This looks good to me and I'm happy to support the change to tailwind :+1: Do we need buy-in/approval from anyone in particular to go ahead with it (your issue mentioned previous discussion)?

Had been thinking of doing the exact same sizing options for the spinner so thanks for that too :blush:

File: frontend/tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  important: true,
+  content: ["./pages/**/*.{js,ts,jsx,tsx}", "./src/**/*.{js,ts,jsx,tsx}"],
+  theme: {
+    extend: {
+      colors: {
+        primary: "var(--bg-color-primary)",
```
Comment (razzeee) [inline frontend/tailwind.config.js:7]: not too happy with the names

File: frontend/tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  important: true,
```
Comment (razzeee) [inline frontend/tailwind.config.js:2]: should only be needed until we phase out all `*.modules.scss` files

**Comment by razzeee** [comment]
- Shadows are also slightly different

I think we want buyin from @barthalion as he basically blocked it last time.

@sorin-davidoi  would this help fix #300 ?

**Comment by barthalion** [comment]
> I think we want buyin from @barthalion as he basically blocked it last time.

I don't remember the details and I don't particularly care as long as it looks good to everyone.

**Comment by razzeee** [comment]
Turns out I misremembered, sorry @barthalion 

https://github.com/flathub/frontend/issues/3#issuecomment-946896611

Turns out I meant @bilelmoussaoui

**Comment by sorin-davidoi** [comment]
> @sorin-davidoi would this help fix https://github.com/flathub/website/issues/300 ?

I think so, as it removes all the inline styles.

**Pull request merged**